### PR TITLE
Pass the namelist variables from the dycore to the physics during the initialization

### PR DESCRIPTION
**Pull request by linjiongzhou**
**Description**
The dynamical core and physics package share some namelist variables. In the current model codebase, these namelist variables are either passed from the dynamical core to the physics package after the initialization or duplicated in the physics package. This produces a potential issue that the physics initialization does not see the dynamical core namelist and leads to errors.

Fixes # (issue)
No issue with this atmos_drivers. We need some code changes in atmos_drivers so that our update of the dynamical core and physics package will work.

**How Has This Been Tested?**
It was tested and the results are expected. The issue has been resolved.

**Checklist:**
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- My changes generate no new warnings
- Any dependent changes have been merged and published in downstream modules
- [ ] New check tests, if applicable, are included

### CODE REVIEW

**Comment by linjiongzhou** [comment]
Hi @laurenchilutti, this merge will only work with the recent updates in the atmos_cubed_sphere and fv3_gfsphysics. Those updates are under review by @spencerkclark and @lharris4.

**Review by spencerkclark — APPROVED** [review]
Thanks @linjiongzhou -- indeed this is very clean!

File: SHiELD/atmos_model.F90 [hunk 1]
```diff
@@ -473,6 +475,10 @@ subroutine atmos_model_init (Atmos, Time_init, Time, Time_step, iau_offset)
    Init_parm%xlat            => Atmos%lat
    Init_parm%area            => Atmos%area
    Init_parm%tracer_names    => tracer_names
+!--- setup IPD_Control
```
Comment (lharris4) [inline SHiELD/atmos_model.F90:478]: This is indeed very clean.

**Review by lharris4 — APPROVED** [review]

**Comment by lharris4** [comment]
Hi,all. Did this ever get merged?

**Comment by laurenchilutti** [comment]
No,  this hasnt been merged.  I plan to merge this at the same time as the public release code (which should be coming this week)

**Comment by lharris4** [comment]
Hi, Lauren. Thanks for your update.

Lucas

**Pull request merged**